Installer/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -eu
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo 'root privileges required' >&2
+    exit 1
+fi
+
+cd "$(dirname "$0")"
+
+[ -d /opt ] || install -d -m 755 -o root -g root /opt
+cp -R --preserve=mode,timestamps itgmania /opt
```

Comment:
Just to confirm, how does this work for upgrades? Since we would likely want to reinstall Simply Love as well as potentially noteskins if we upgrade them

Reply:
That's a good question. I didn't think about update. This would overwrite existing files, but not delete files that don't exist anymore in the new release. I guess that might be a problem for SL?

Reply:
Yes that would be a problem for SL (which is why SL docs recommend deleting the original theme and completely replacing it). Maybe the script can capture that (look for an existing `Simply Love` folder in themes, if it exists then delete it).

I'm looking to also update some of the noteskins so we might need to do the same thing for those.